src/app/shell/personal-cabinet/personal-cabinet.component.html

```diff
@@ -1,16 +1,58 @@
 <div class="container">
-  <h1 class="title">{{userRole === 'provider' ? 'КАБІНЕТ НАДАВАЧА' : userRole === 'admin' ? 'КАБІНЕТ ТЕХ. АДМІНА' : 'КАБІНЕТ КОРИСТУВАЧА'}}</h1>
+  <h1 class="title">
+    {{
+      userRole !== "provider"
+        ? PersonalCabinetTitle[userRole]
+        : PersonalCabinetTitle[subrole$ | async]
```

Comment:
I think you should chekc for subRole
subrole ? PersonalCabinetTitle[subrole ] : PersonalCabinetTitle[userRole]
if no subrole, then it will use provider for enum. do the subscription in this case in the component, so you won't have to async in the template